bundlewrap/cmdline/debug.py

```diff
@@ -1,13 +1,13 @@
+import readline
```

Comment:
There's also `gnureadline`, but using that will break tab completion. I think this is the best we can do.

Should we add a warning for apple users that only the history of the last closed `bw debug` shell will get saved?

Reply:
I guess adding a warning for unintended behaviour where the only feasible solution is "buy another computer" is kinda pointless.